### PR TITLE
Propagate changes made to fink-broker (stream all alert data)

### PR DESCRIPTION
**Pull request by cAbhi15**
**IMPORTANT: Please create an issue first before opening a Pull Request.**
Linked to issue(s): #19 

## What changes were proposed in this pull request?
The pull request at [fink-broker/#269](https://github.com/astrolabsoftware/fink-broker/pull/269) allows the distribution of whole alerts (original data + added value by fink).

This PR propagates those changes to `fink-client`. The following are the list of additions in this PR:

- [x] modify sample avro alerts to include all data (for tests)
- [x] add a jupyter notebook illustrating how the alerts can be viewed (plotting cutout stamps)
- [x] update documentation

## How was this patch tested?
manually, CI

### CODE REVIEW

**Comment by cAbhi15** [comment]
@JulienPeloton 
For this PR, I have branched out of [cAbhi15/fink-client/tree/issues/16/test_coverage](https://github.com/cAbhi15/fink-client/tree/issues/16/test_coverage) (see PR #17) for keeping an account of the added changes. 

The added changes are from the commit db60733.

**Comment by cAbhi15** [comment]
@JulienPeloton could you review this PR?

File: fink_client/alertUtils.py [hunk 1]
```diff
@@ -15,112 +15,83 @@
...
-import confluent_kafka
 import json
 import glob
-import time
 import fastavro
 from typing import Iterable
 
 def read_avro_alerts(data_path: str) -> Iterable[dict]:
-    """ Read avro alert files and return an interable 
+    """ Read avro alert files and return an interable
```
Comment (JulienPeloton) [inline fink_client/alertUtils.py]: Typo: iterable

File: fink_client/alertUtils.py [hunk 2]
```diff
@@ -15,112 +15,83 @@
...
     Parameters
     ----------
     data_path: str
         a directory path where to look for avro alert files
-        
+
     Returns
     ----------
     record: Iterable
         a generator that yields records(dict) after reading avro files
```
Comment (JulienPeloton) [inline fink_client/alertUtils.py]: Typo: there should be a space in between records and dict?

File: tests/test.py [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import unittest
+import os
+import io
+import confluent_kafka
+import time
+import fastavro
+from fink_client.consumer import AlertConsumer
+from fink_client.alertUtils import (
```
Comment (JulienPeloton) [inline tests/test.py]: I would rather use:
```python
from fink_client.alertUtils import read_avro_alerts
from fink_client.alertUtils import encode_into_avro
from fink_client.alertUtils import get_legal_topic_name
```

File: README.md [hunk 4]
```diff
@@ -108,3 +106,5 @@ with AlertConsumer(topics, config) as consumer:
     topic, alert = consumer.poll(5)
     print("topic: {}, alert:\n {}".format(topic, alert))
 ```
+
+To learn more about the schema of received alerts and how to work with them, refer to the [jupyter notebook](notebooks/working_with_alerts.ipynb).
```
Comment (JulienPeloton) [inline README.md]: Could you add WIP at the end of the sentence? I will update it in a future PR.

File: fink_client/alertUtils.py (hunk 1 above)
Comment (cAbhi15) [reply fink_client/alertUtils.py]: fixed in 7be307a.

File: fink_client/alertUtils.py (hunk 2 above)
Comment (cAbhi15) [reply fink_client/alertUtils.py]: fixed in 7be307a.

File: tests/test.py (hunk 3 above)
Comment (cAbhi15) [reply tests/test.py]: fixed in 7be307a.

File: README.md (hunk 4 above)
Comment (cAbhi15) [reply README.md]: done in 7be307a.

**Comment by cAbhi15** [comment]
Thanks @JulienPeloton for the review. I have made the suggested changes. Could you merge it?

**Comment by JulienPeloton** [comment]
Merging! Thanks @cAbhi15

**Pull request merged**